ripe/atlas/cousteau/__init__.py

```diff
@@ -158,6 +161,11 @@ def build_url(self):
             self.build_url_chunks()
             return self.split_urls.pop(0)
 
+        # Reduce datetimes to UNIX timestamps
+        for k, v in self.api_filters.items():
+            if isinstance(v, datetime):
+                self.api_filters[k] = int(time.mktime(v.timetuple()))
+
         filters = '&'.join("%s=%s" % (k, v) for (k, v) in self.api_filters.items())
```

Comment:
Can you use following way instead:
```python
import calendar
int(calendar.timegm(self.start.timetuple()))
```
The diff is that the above version will assume UTC, but yours will assume users local.
I founded out myself recently and I change it accordingly.